itou/www/autocomplete/views.py

```diff
@@ -54,3 +54,19 @@ def jobs_autocomplete(request):
         ]
 
     return HttpResponse(json.dumps(appellations), "application/json")
+
+
+def prescriber_authorized_organizations_autocomplete(request):
+    term = request.GET.get("term", "").strip()
+
+    organizations = (
+        [
+            {"value": org.name, "id": org.id}
+            for org in PrescriberOrganization.objects.autocomplete(term)
+            if org.is_authorized and org.kind != PrescriberOrganization.Kind.PE
```

Comment:
À la limite tu peux peut-être restreindre la query tout de suite plutôt que d'itérer sur le tout :

```python
            for org in PrescriberOrganization.objects.exclude(
                is_authorized=False, kind=PrescriberOrganization.Kind.PE
            ).autocomplete(term)
```

Reply:
Merci, c'est aussi l'occase de rajouter un test sur la partie autocomplete des prescripteurs/orga

Reply:
2 params dans l'exclude, ca fait un AND ou un OR ?

Reply:
Ça fait un `AND` tu peux toujours le vérifier avec :

```python
a = PrescriberOrganization.objects.exclude(is_authorized=False, kind=PrescriberOrganization.Kind.PE)
print(a.query)
```

Reply:
thx